src/components/TrackView.js

```diff
@@ -1,8 +1,9 @@
 import React, {Component} from "react";
-import {Header} from "semantic-ui-react";
+import {Search} from "semantic-ui-react";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'Search'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)